### ISSUE TITLE
Status updates and screenshots

### ISSUE THREAD

**Issue by xycloud** [ppadovani/KibanaNestedSupportPlugin#8]
would you mind to provide a screenshot of this plugin?

**Comment by ppadovani** [comment]
This plugin is still a work in progress. I posted a screenshot in another issue that I then closed, however I'll leave this issue open and post screenshots and updates as the development progresses.

- With the plugin installed, you must activate nested support for each specific index pattern. There is a new management option for index patterns that will show up.
<img width="1486" alt="screen shot 2017-12-15 at 7 36 41 am" src="https://user-images.githubusercontent.com/11234684/34044743-dbb6b392-e16b-11e7-9432-0a62ef868ef9.png">

This will list out the configured index patterns, and allow you to activate/deactivate nested support.
<img width="1481" alt="screen shot 2017-12-15 at 7 37 07 am" src="https://user-images.githubusercontent.com/11234684/34044775-fce070a8-e16b-11e7-8c6a-6e04a033f213.png">

- At this time the only other feature that has been ported from my fork of Kibana is the query parser that generates nested queries from a SQL like language. In this shot you can see a query that leverages both non-nested and nested fields, as well as the results.
<img width="1475" alt="screen shot 2017-12-15 at 7 43 55 am" src="https://user-images.githubusercontent.com/11234684/34044824-34dcef5e-e16c-11e7-9a7a-743c8f327855.png">

At this point I'm working on cleaning up error handling for the discover application for no results, and query syntax errors. Once that is complete I'll move on to working on the aggregation support.

**Comment by ppadovani** [comment]
The no results found will now display different text depending upon the nested state of an index pattern. Here is what no results looks like for an index pattern that has nested turned on:
<img width="1434" alt="screen shot 2017-12-15 at 9 42 11 am" src="https://user-images.githubusercontent.com/11234684/34049263-626377b2-e17c-11e7-8041-8de9adda8210.png">

**Comment by ppadovani** [comment]
Small update: Added the syntax feedback on queries:

<img width="896" alt="screen shot 2017-12-23 at 10 31 15 am" src="https://user-images.githubusercontent.com/11234684/34321101-a5c34ffe-e7cc-11e7-8afa-28825c23c0aa.png">

**Comment by xycloud** [comment]
“I'm going to start work on the aggregation support now”
did you mean show agg result on discover page?

**Comment by ppadovani** [comment]
Small update... was able to get the nested aggregations partially working.. some bugs I need to deal with and clean up. I still think I'm on track for a beta release by the end of the year.

**Comment by ppadovani** [comment]
@xycloud - No... the actual Visualizations that use aggregations to pull data buckets back from elastic search. The code that is checked in does properly create nested search queries for elastic search and return results that contain nested fields. The nested results that show up on the discover page will currently still show as JSON blobs, support for displaying nested objects in a sane way will come later. There are some things I need to think about on how to approach the problem from this plugin perspective.

**Comment by ppadovani** [comment]
Holiday update, aggregations seem to be working... still have testing and debugging to do:

<img width="1623" alt="screen shot 2017-12-26 at 9 54 55 am" src="https://user-images.githubusercontent.com/11234684/34360057-ddf0a5ae-ea22-11e7-96da-bba352c3a643.png">

**Comment by ppadovani** [comment]
I have merged in support for nested aggregations. The remaining work is around the discovery results and how to display nested objects.

**Comment by ppadovani** [comment]
I've gone ahead and released a pre-release of 1.0.0 with support for Kibana 5.6.5. This includes all of the above support and may still have defects kicking around in it.

[V1.0.0](https://github.com/ppadovani/KibanaNestedSupportPlugin/releases/tag/v1.0.0)

**Comment by ppadovani** [comment]
Added documentation to the README, and updated the no results page to link to the README documentation.

**Comment by ppadovani** [comment]
Updated the release to beta2 to fix my breaking timelion.

**Comment by ppadovani** [comment]
Update: I'm working on #1 and am taking a new approach than what the my fork of Kibana had done. I wanted to see if anyone had any feedback on this approach. Please see #1 for details, and please leave comments.

**Comment by ppadovani** [comment]
This plugin is feature complete. I have a few cleanup tasks I want to perform on the code which should take a few days. Once that is done, I'll cut the 5.6.5-1.0.0 version, then begin forward porting to 6.0.X and 6.1.X. I'll back port after I have forward ported, unless someone asks for priority porting.

**Comment by ppadovani** [comment]
Quick update: I one other thing I want to do for #12, then I want to take a quick evaluation on #18 which is a port of an issue from my old fork. Once I've closed those two issues out, I'll cut and start porting. I anticipate the cut happening this weekend short of any other issues appearing from the previous beta releases.

**Comment by ppadovani** [comment]
Cut and released 5.6.5. Working on porting to other versions.
